pkgs/by-name/lu/lutgen-studio/package.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  makeWrapper,
+
+  fontconfig,
+  libGL,
+  libxkbcommon,
+  openssl,
+  wayland,
+  xorg,
+
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lutgen-studio";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "ozwaldorf";
+    repo = "lutgen-rs";
+    tag = "${pname}-v${version}";
+    hash = "sha256-Oog3qv+Pvo/Ib/fpoIcrExVkkcZj+tX8unH9LZod3YM=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-+8Sq6Jk914U0Vyr8cKLX++AF41j0R7y76ZG96SZJtxM=";
+
+  cargoBuildFlags = [
+    "--bin"
+    "${pname}"
+  ];
+  cargoTestFlags = [
+    "-p"
+    "${pname}"
+  ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  # Include dynamically loaded libraries
+  LD_LIBRARY_PATH = "${lib.makeLibraryPath ([
+    fontconfig
+    libGL
+    libxkbcommon
+    openssl
+    wayland
+    xorg.libXcursor
+    xorg.libXrandr
+    xorg.libXi
+    xorg.libX11
+  ])}";
```

The output of `lib.makeLibraryPath` is already a string. also we don't need the `()` around the list
```suggestion
  LD_LIBRARY_PATH = lib.lib.makeLibraryPath[
    fontconfig
    libGL
    libxkbcommon
    openssl
    wayland
    xorg.libXcursor
    xorg.libXrandr
    xorg.libXi
    xorg.libX11
  ];
```